qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
```

Comment:
@squirrelo, you are right, the JSON is dynamic. However, as stated in my previous comment the AJAX call is not executed as it is in JS. Thus, the page never "renders" the study list == no way to test in here. The only option is to add tests to /admin/portals/studiesAJAX/, which is way beyond the issue this PR is solving but I could do it.